microsoftcognitiveservices/src/test/java/science/aist/imaging/service/microsoftcognitiveservices/SpecialSpringConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021 the original author or authors.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package science.aist.imaging.service.microsoftcognitiveservices;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import science.aist.imaging.api.domain.wrapper.ImageFactory;
+import science.aist.imaging.api.domain.wrapper.implementation.ImageFactoryFactory;
+
+import java.awt.image.BufferedImage;
+
+/**
+ * <p>Helper class for creating special java beans</p>
+ *
+ * @author Andreas Pointner
+ */
+@Configuration
```

Comment:
Yeah, I know it is my code, but the `@since` tag is missing. For tests this would probably be ok, but this is a "main" class.

Reply:
Fixed in [739c19f](https://github.com/FHOOEAIST/imaging/pull/14/commits/739c19fa2d18dad40ebac4676254c36ff07279f0)